cuml/random_forest_demo.ipynb

```diff
@@ -0,0 +1,286 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Random Forest and Pickling\n",
+    "The Random Forest algorithm is a classification algorithm which builds several decision trees, and aggregates each of their outputs to make a prediction.\n",
+    "\n",
+    "In this notebook we will train a scikit-learn and a cuML Random Forest Classification model. Then the cuML model is pickled and loaded for prediction. We also compare the results of the scikit-learn, non-pickled and pickled cuML models.\n",
```

Comment:
How about changing the second sentence to `Then we save the cuML model for future use with Python's "pickling" mechanism and demonstrate how to re-load it for prediction." (Some people may be less familiar with the concept of pickling)